wisp/wisp-feature-testing/src/main/kotlin/wisp/feature/testing/FakeLegacyFeatureFlags.kt

```diff
@@ -369,8 +380,9 @@ constructor(
         value: T,
         attributes: Attributes = defaultAttributes
     ) {
-        val jsonValue = { moshi().adapter(T::class.java).toSafeJson(value) }
-        overrideKey(feature, key, jsonValue, attributes)
+        val m = moshi().adapter(T::class.java)
```

Comment:
```suggestion
        val adapter = moshi().adapter(T::class.java)
```

Adapter is a more idiomatic variable name and leaves the code more readable.